cmd/run.go

```diff
@@ -85,12 +85,22 @@ func runRun(ctx context.Context, ro options.RunOptions, args []string, signers .
 		attestors = append(attestors, commandrun.New(commandrun.WithCommand(args), commandrun.WithTracing(ro.Tracing)))
 	}
 
-	addtlAttestors, err := attestation.Attestors(ro.Attestations)
-	if err != nil {
-		return fmt.Errorf("failed to create attestors := %w", err)
+	for _, a := range ro.Attestations {
+		for _, att := range attestors {
+			if a == att.Name() {
+				log.Warnf("Attestator %s already declared, skipping", a)
+				break
+			} else {
+				attestor, err := attestation.AddAttestor(a)
+				if err != nil {
+					return fmt.Errorf("failed to create attestor: %w", err)
```

Comment:
Do we want to fail at the first attestor, or would it be better to continue trying to add the subsequent attestors and give the user an error with all failed attestors?
My question is, because we do it for duplicated ones, we warn the user. Is there any reason to fail at the first attestors?

Example: Let's say the user sends attestors `a, b, c, d, e, f` 
`a` and `c` are duplicated
`d` return error.
`b`, `e`, `f` are good.

We will warn the user about `a`, include `b`, warn the user about `c`, and fail in `e`,

Feel free to ignore if I'm missing some context here :)

Reply:
hmmm, so I don't feel *really* strongly either way, but I suppose you're asking 'if failing to create an attestor fails, should we fail?'.

If the `AddAttestor` function fails, I think we should stop. I don't think a `witness run` should continue if not all the expected attestations are going to be generated. I think that I am confident that the command should fail on failed attestor creation.

Going back to the original problem that this PR removes, I tackled this by warning the user that they have repeated themselves / declared an attestor that has already been declared (e.g., by default). Possibly it could be a better practice to just fail in this situation. After all, it may be better to be simple and say "you've duplicated attestor defintion, please fix the invocation".

Reply:
@jkjell @mikhailswift, any thoughts?

Reply:
To throw a 🔧 in the conversation, I create https://github.com/in-toto/witness/issues/340 to capture thoughts around a future idea that may be counter to this one.

Reply:
@jkjell I think that what you mention in #340 makes sense, but I don't think it collides too much with this due to the fact that (at least for now) there is no reason why you would want to run an attestor more than once. I think that we should merge this for now, and we could potentially change it in future if/when we implement #340.